### PR TITLE
changed the method of getting caliper logging path

### PR DESCRIPTION
**Pull request by HamzaIbnFarooq**
On servers `default user` don't have permissions to `read/write logs`, thus we need to implement the same methodology as implemented by `edx tracking`.
So I used `syslog` to get the logs separately written in another file.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -40,7 +40,15 @@ these files should be located at ``/edx/app/edxapp/`` directory::
...
-Transformed event logs can be found in ``/edx/var/logs/tracking/tracking.log`` file.
+Transformed events are logged using **'logging.handlers.SysLogHandler'** with **'facility: local2'**.
+
+To get logs, follow these steps:
+
+1. Create a log file with read/write permissions given to **syslog** user
+
+2. Create a mapping for **'local2'** in the configuration files present in **/etc/rsyslog.d/** ::
+
+    local2.*                 /path/of/file.log
```
Comment (asamolion) [inline README.rst]: Shouldn't we specify a default path for the logging file?

File: openedx_caliper_tracking/loggers.py [hunk 2]
```diff
@@ -8,14 +8,13 @@
...
     """Return a formatted log message"""
     return '{} [{}][event_type:{}] Returned status: {}'.format(
-                result, filename, event_name, status_code
-            )
+        result, filename, event_name, status_code
+    )
 
 
-def get_caliper_logger(logger_name, file_path):
+def get_test_logger(logger_name, file_path):
```
Comment (asamolion) [inline openedx_caliper_tracking/loggers.py:15]: is the name `get_test_logger` appropriate?

File: openedx_caliper_tracking/script.py [hunk 3]
```diff
@@ -14,7 +14,7 @@
 TESTS_LOG_FILE = 'caliper_tests.log'
 
 
-logger = get_caliper_logger('caliper_tests', TESTS_LOG_FILE)
+logger = get_test_logger('caliper_tests', TESTS_LOG_FILE)
```
Comment (asamolion) [inline openedx_caliper_tracking/script.py:17]: What is the purpose of the test logger?

File: openedx_caliper_tracking/settings.py [hunk 4]
```diff
@@ -16,4 +16,4 @@
 CALIPER_AUTH = {
     'CALIPER_TESTS_API_URL': '',
```
Comment (asamolion) [inline openedx_caliper_tracking/settings.py:17]: Shouldn't these be loaded from the settings module?

**Review by asamolion — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (HamzaIbnFarooq) [reply README.rst]: We can but I don't think we need to, any given path with the syslog permissions will work for this plugin.

File: openedx_caliper_tracking/loggers.py (hunk 2 above)
Comment (HamzaIbnFarooq) [reply openedx_caliper_tracking/loggers.py:15]: The caliper test script logs using this logger. if you have any better name kindly suggest.

File: openedx_caliper_tracking/script.py (hunk 3 above)
Comment (HamzaIbnFarooq) [reply openedx_caliper_tracking/script.py:17]: The caliper test script logs using this logger

File: openedx_caliper_tracking/settings.py (hunk 4 above)
Comment (HamzaIbnFarooq) [reply openedx_caliper_tracking/settings.py:17]: yes that can be done, but its a unit test. So i think we shouldn't.

File: README.rst (hunk 1 above)
Comment (asamolion) [reply README.rst]: Yes, but we should recommend a default path anyways. `/edx/var/log/caliper/caliper.log` will suffice.

File: openedx_caliper_tracking/loggers.py (hunk 2 above)
Comment (asamolion) [reply openedx_caliper_tracking/loggers.py:15]: Okay, that works.

File: openedx_caliper_tracking/script.py (hunk 3 above)
Comment (asamolion) [reply openedx_caliper_tracking/script.py:17]: Oka.

File: README.rst (hunk 1 above)
Comment (HamzaIbnFarooq) [reply README.rst]: ok.

File: openedx_caliper_tracking/settings.py (hunk 4 above)
Comment (asamolion) [reply openedx_caliper_tracking/settings.py]: You should. There is a test config present in Open edX. Please add code and documentation to get these vars from the config.

Comment (asamolion) [reply openedx_caliper_tracking/settings.py:17]: Okay, got it. thanks.

**Review by asamolion — APPROVED** [review]

**Review by imhassantariq — APPROVED** [review]
Manual QA Done. :100: :+1:

**Pull request merged**